extensions/wikia/Email/EmailController.class.php

```diff
@@ -34,6 +38,7 @@ public function init() {
 			$this->currentUser = $this->findUserFromRequest( 'currentUser', $this->wg->User );
 			$this->targetUser = $this->findUserFromRequest( 'targetUser', $this->wg->User );
 			$this->test = $this->getRequest()->getVal( 'test', false );
+			$this->marketingFooter = $this->request->getVal( 'marketingFooter' );
```

Comment:
That would make me happy :) At the very least because our default value of `$marketingFooter = false;` is lost as soon as we make this `$this->marketingFooter = $this->request->getVal( 'marketingFooter' );` call. I do also like the explicitness.